CMSgov/cms-carts-seds
# [MDCT-2923] Section H with new part 2

### Description

- Adds new Part 2
- Updates existing part 2 -> part 3
- Updates text overall

### Related ticket(s)


MDCT-2923

---
### How to test

- Compare the diffs
- pull it down
- release 2022 and 2023
- Try section 1 q 1 Yes and no, check for conditional rendering
- Compare against request

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
